Objective-C/CBLReplicatorConfiguration.m

```diff
@@ -145,32 +148,61 @@ - (void) setNetworkInterface: (NSString*)networkInterface {
     _networkInterface = networkInterface;
 }
 
+- (CBLCollectionConfiguration*) defaultCollectionConfig {
+    CBLCollection* defaultCollection = [_database defaultCollectionOrThrow];
+    return _collectionConfigs[defaultCollection];
+}
+
+- (CBLCollectionConfiguration*) defaultCollectionConfigOrThrow {
+    CBLCollectionConfiguration* config = [self defaultCollectionConfig];
+    if (!config) {
+        [NSException raise: NSInternalInconsistencyException
+                    format: @"Attempt to perform action with empty default collection config"];
```

Comment:
It is not "empty default collection config". Maybe "No default collection added to the configuration."